### PR TITLE
Some Updates

### PR DESCRIPTION
**Pull request by Jeff1999**

### CODE REVIEW

File: 100_Numpy_exercises_with_hints_with_solutions.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ print(Z)
 `hint: reshape`
 
 ```python
-nz = np.nonzero([1,2,0,0,4,0])
+nz = np.nonzero([1,2,0,0,4,0]).reshape(3, 3)
```
Comment (rougier) [inline 100_Numpy_exercises_with_hints_with_solutions.md:75]: I think there's a confusion with the next question.

File: 100_Numpy_exercises_with_hints_with_solutions.md [hunk 2]
```diff
@@ -463,7 +463,7 @@ for dtype in [np.float32, np.float64]:
 `hint: np.set_printoptions`
 
 ```python
-np.set_printoptions(threshold=np.nan)
+np.set_printoptions(threshold=float("inf"))
```
Comment (rougier) [inline 100_Numpy_exercises_with_hints_with_solutions.md:466]: I think is has been fixed but for some reason it does not appear in this file.

**Comment by rougier** [comment]
Thanks, but I think the file you've edited is out dated. There's now a generic file that is used to produce the md and notebook files.

**Comment by Jeff1999** [comment]
Nah Sir...&nbsp;
I've forked your latest md file (100_Numpy_exercises_with_hints_with_solutions.md) and it still doesn't work on NumPy (my version 1.18.5).&nbsp;
But I haven't try it on latest version (1.19) updated on Jun 29 though. Hope it works now ^^



------------------&nbsp;Original&nbsp;------------------

**Comment by rougier** [comment]
Sorry, I was not clear. You need to modify this file: https://github.com/rougier/numpy-100/blob/master/source/exercises100.ktx and make sure your copy is up to date (your PR has conflict). Then you can modify the inf/nan problem (I was referring to your other fix in your PR).

**Pull request closed without merging**